sentry-core/src/main/java/io/sentry/core/Sentry.java

```diff
@@ -86,12 +90,12 @@ public static void init() {
    * @throws InvocationTargetException the InvocationTargetException
    */
   public static <T extends SentryOptions> void init(
-      @NotNull OptionsContainer<T> clazz,
-      @NotNull OptionsConfiguration<T> optionsConfiguration,
-      boolean globalHubMode)
+      final @NotNull OptionsContainer<T> clazz,
```

Comment:
Super hard to review this PR with so much unrelated changes like this

Reply:
true, I should not have changed this, but it's a Draft right